database-restore-postgres.sh

```
@@ -0,0 +1,289 @@
+    list_backups
+    exit 0
+fi
+
+# Get backup to restore
+if [[ "$use_latest" == "true" ]]; then
+    backup_file=$(get_latest_backup)
+    [[ -z "$backup_file" ]] && handle_error "No backups found for database '$database'"
+    log "Using latest backup: $(basename "$backup_file")"
+elif [[ -z "$backup_file" ]]; then
+    list_backups
+    read -p "Enter backup name to restore: " backup_choice
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)